tty.go

```diff
@@ -28,6 +30,211 @@ func (t *tty) copyIO(w io.Writer, r io.ReadCloser) {
 	r.Close()
 }
 
+type EpollConsole struct {
```

Comment:
Can we maybe move this into a package that can be imported as every client that does detach will have to implement this.